### PR TITLE
feat: [CDS-47126]: force delete env

### PR DESCRIPTION
**Pull request by RajBaviskar**
#### Screenshots

https://user-images.githubusercontent.com/7086440/213124115-40454d6a-8c37-4897-b2cf-32f5a5b85952.mov

https://user-images.githubusercontent.com/7086440/213124467-f778349b-bd40-4916-bb26-fb26d09ec026.mov

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by RajBaviskar** [comment]
retrigger coverage

**Comment by RajBaviskar** [comment]
retrigger jest

**Comment by RajBaviskar** [comment]
retrigger sonar

**Comment by RajBaviskar** [comment]
retrigger jest

File: src/modules/75-cd/components/EnvironmentsV2/EnvironmentsList/__test__/EnvironmentsList.test.tsx [hunk 1]
```diff
@@ -138,4 +139,38 @@ describe('EnvironmentsList', () => {
     fireEvent.click(getByText(form, 'delete') as HTMLButtonElement)
     await waitFor(() => expect(mockErrorHandler).toHaveBeenCalled())
   })
+
+  test('Error handling during the deletion of items from the Environment Menu for force ref entity', async () => {
+    jest.spyOn(FeatureFlag, 'useFeatureFlags').mockReturnValue({
```
Comment (AshwinBhatkal-Harness) [inline src/modules/75-cd/components/EnvironmentsV2/EnvironmentsList/__test__/EnvironmentsList.test.tsx:144]: We should not be doing this. TestWrapper has a prop that we can use to set feature flags

**Review by AshwinBhatkal-Harness — CHANGES_REQUESTED** [review]
Move the feature flag mock as testWrapper props

**Comment by RajBaviskar** [comment]
> Move the feature flag mock as testWrapper props
changed.

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by PratyushGar — APPROVED** [review]

**Comment by RajBaviskar** [comment]
retrigger cypress-cd

**Review by vkbansal-harness — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**